docs/SUMMARY.md

```diff
@@ -135,7 +131,10 @@
         * [Travis CI](guides/ci/travis.md)
         * [CircleCI](guides/ci/circleci.md)
     * [Static Linking](guides/static_linking.md)
-* [Platform Support](platform_support.md)
+    * [Database](database/README.md)
+        * [Connection](database/connection.md)
+        * [Connection pool](database/connection_pool.md)
+        * [Transactions](database/transactions.md)
 * [Tutorials](tutorials/README.md)
```

Comment:
I'd delete the index page for tutorials, it doesn't have any useful content
```suggestion
* Tutorials
```

Reply:
Not *yet*. There are already plans for follow-up courses... But yeah we can leave the tutorials landing page out as long es it has no meaningful content.

Reply:
*tutorials/basics/README.md* already looks like it wants to be the landing page, so let it!

Reply:
Oh still need to actually delete the page

> The following pages exist in the docs directory, but are not included in the "nav" configuration:
>  - tutorials/README.md